.github/workflows/main.yml

```diff
@@ -1,21 +1,14 @@
-name: CI
-
+name: Publish Docker
 on:
   push:
     branches:
       - stable
-
 jobs:
   build:
-
     runs-on: ubuntu-latest
-    
     steps:
     - name: checkout stable branch
       uses: actions/checkout@stable
-    - name: build container
-      uses: "actions/docker/cli@master"
-      args: "build -t caluma-demo ./caluma-demo"
```

Comment:
Why is building the container not necessary anymore?

Reply:
As far as I now this is done automatically with the build keyword on line 6. https://github.com/marketplace/actions/build-tag-publish-docker